ext_emconf.php

```diff
@@ -17,7 +17,7 @@
     'category' => 'misc',
     'constraints' => [
         'depends' => [
-            'php' => '8.1.0-8.3.99',
+            'php' => '8.1.0-8.4.99',
```

Comment:
```suggestion
            'php' => '8.1.0-8.3.99',
```
Again, we do not officially support PHP 8.4 as long as we still support TYPO3v11. Please adjust this locally if need arises.

Reply:
TYPO3 v11 will still limit the PHP releases to 8.3, even if the dlf extension raises the limit. So for TYPO3 v11 nothing changes. The important point is that the new limit works with TYPO3 v12.

Reply:
OK, fine. Thanks for the reminder!